### PR TITLE
Shippable updates

### PR DESCRIPTION
**Pull request by jforissier**

### CODE REVIEW

File: .shippable.yml [hunk 1]
```diff
@@ -33,11 +33,11 @@ build:
     - _make CFG_WITH_PAGER=y
     - _make CFG_WITH_PAGER=y CFG_WITH_LPAE=y
     - _make CFG_WITH_LPAE=y
-    - _make CFG_WITH_STATS=y
     - _make CFG_RPMB_FS=y
     - _make CFG_RPMB_FS=y CFG_RPMB_TESTKEY=y
     - _make CFG_REE_FS=n CFG_RPMB_FS=y
-    - _make CFG_WITH_USER_TA=n CFG_CRYPTO=n CFG_SE_API=n CFG_PCSC_PASSTHRU_READER_DRV=n
+    - _make CFG_WITH_USER_TA=n CFG_CRYPTO=n CFG_SE_API=n
```
Comment (jenswi-linaro) [inline .shippable.yml]: As these targets aren't executed here, why do we need to build `CFG_PCSC_PASSTHRU_READER_DRV=y` separatebly?

Comment (jforissier) [reply .shippable.yml]: No particular reason. Updated to simply replace `=n` with `=y`.

**Comment by jforissier** [comment]
Pushed one more commit to remove deprecated flags.

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Pull request merged**